README.md

```diff
@@ -60,6 +60,7 @@ The MCP Inspector supports the following configuration settings. To change them,
 | `MCP_REQUEST_TIMEOUT_RESET_ON_PROGRESS` | Reset timeout on progress notifications                                                                      | true    |
 | `MCP_REQUEST_MAX_TOTAL_TIMEOUT`         | Maximum total timeout for requests sent to the MCP server (ms) (Use with progress notifications)             | 60000   |
 | `MCP_PROXY_FULL_ADDRESS`                | Set this if you are running the MCP Inspector Proxy on a non-default address. Example: http://10.1.1.22:5577 | ""      |
+| `MCP_SERVER_URL`                        | Set this to configure the default MCP Server to connect to on startup. Example: http://localhost:8080        | ""      |
```

Comment:
http://localhost:8080/sse

Reply:
* It would be _nice_ if this could be `MCP_SERVER_BASE_URL`, and keep the example `http://localhost:8080`. 
* You would need to add `/sse` to it in the UI before passing it to the sidebar. 
* Otherwise, this needs to be the full url to an endpoint, not just the server and port. 
* Note soon the `/sse` part will need to become `/mcp`.